### PR TITLE
CH-195: Node configuration screen in configuration section

### PR DESCRIPTION
**Pull request by NareshGopal**
# Problem Statement
- Develop the node configuration screen in the configuration section

# Design
-  Develop the node configuration screen as per the [design](https://www.figma.com/file/famlEpyBra0P5RY1SWpmSa/Node---Configuration-Screen?node-id=73%3A210)

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by ajaysrivas** [comment]
Reviewed by looking at actual GUI. The code review will be done by Sujit once he is back from vacation.

File: src/components/configuration/LrNodeConfiguration.vue [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+  }
+
+  clearPassword() {
+    this.password = {
+      newPassword: "",
+      confirmNewPassword: "",
+    };
+  }
+
+  @Watch("selectedNode")
```
Comment (sujit-h-singh) [inline src/components/configuration/LrNodeConfiguration.vue:311]: emit change event directly

File: src/components/configuration/LrNodeConfiguration.vue [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+      searchDomains,
+    };
+    this.initialNetworkValues = { ...this.network };
+  }
+
+  applyNetworkInfo() {
+    //API call to modify the network data
+  }
+
+  resetNetworkData() {
```
Comment (sujit-h-singh) [inline src/components/configuration/LrNodeConfiguration.vue:296]: reset network functionality is not implemented correctly.

File: src/components/configuration/LrNodeConfiguration.vue [hunk 3]
```diff
@@ -0,0 +1,323 @@
+<!--
```
Comment (sujit-h-singh) [inline src/components/configuration/LrNodeConfiguration.vue:1]: isFormValid error on page

**Pull request merged**